identity-did/src/verification/verification_method.rs

```diff
@@ -132,3 +132,11 @@ impl<T> AsRef<CoreDIDUrl> for VerificationMethod<T> {
     self.id()
   }
 }
+
+impl<T> KeyComparable for VerificationMethod<T> {
+  type Key = CoreDIDUrl;
+
+  fn key(&self) -> &Self::Key {
+    self.id()
+  }
```

Comment:
Super nit: You added `#[inline]` in the impl for `MethodRef` but not in other places. Not sure if it's really necessary, but would be nice to be consistent.

Reply:
I added it to `MethodRef` specifically because almost all implementations for `MethodRef` were `#[inline]` but not for other structs. I'll change it for `KeyComparable` though.